### PR TITLE
fix: add modules, camelCase

### PR DESCRIPTION
**Pull request by mafewtm**
## Description

Add the core module and rename some functions to follow the camelCase rule

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

**Comment by MafewTM** [comment]
Not sure why its showing some changes that were already committed.

**Comment by mafewtm** [comment]
fixed whatever the weird issue was where it showed previous changes already committed

File: client.lua [hunk 1]
```diff
@@ -25,16 +25,24 @@ local function pickProcess()
...
+        RequestAnimDict(dict)
+        Wait(1)
+    end
+end
+
 local function PickAnim()
-    lib.requestAnimDict('amb@prop_human_bum_bin@idle_a')
-    TaskPlayAnim(cache.ped, 'amb@prop_human_bum_bin@idle_a', 'idle_a', 6.0, -6.0, -1, 47, 0, 0, 0, 0)
+    local ped = cache.ped
+    LoadAnim('amb@prop_human_bum_bin@idle_a')
```
Comment (mockdot) [inline client.lua]: Why switch to this if lib.requestAnimDict already does the wait for animation load? https://github.com/overextended/ox_lib/blob/master/imports/requestAnimDict/client.lua

**Review by Manason — APPROVED** [review]

File: client.lua (hunk 1 above)
Comment (mafewtm) [reply client.lua]: For some reason is showed previous changes which I already had fixed in my latest commit. My most recent commit fixed that.

**Review by YvanCywan — APPROVED** [review]

**Pull request merged**